roles/common/files/letsencrypt-gencert

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+service apache2 stop
```

Comment:
Should the start of Apache now be deleted from the end of the `letsencrypt.yml` task file?